### PR TITLE
Added saintCamillusDeLellisPriestOrSaintHenryBishopAndMartyr and translations (fixes #115).

### LINKED ISSUE #115: `chile.js`: Fix the celebration on 14 July
Tasks:
1. Remote that comment block.
2. Create a new key (localise it in `en.js`): `"saintCamillusDeLellisPriestOrSaintHenryBishopAndMartyr": "Saint Camillus de Lellis, priest, or Saint Henry, Bishop and Martyr"` (`sk.js`: `Svätého Kamila de Lellis, kňaza, alebo svätého Henricha`).
3. Define `saintCamillusDeLellisPriestOrSaintHenryBishopAndMartyr` celebration in `chile.js`.

### PR DESCRIPTION
**Pull request by jarosz**
Translation based on your comments. I've changed it in poland.js as well.

### CODE REVIEW

File: src/calendars/poland.js [hunk 1]
```diff
@@ -335,17 +335,10 @@ let dates = year => {
       }
     },
     {
-      "key": "saintHenryBishopAndMartyr",
+      "key": "saintCamillusDeLellisPriestOrSaintHenryBishopAndMartyr",
```
Comment (tukusejssirs) [inline src/calendars/poland.js]: @jarosz, I don’t think it is celebrated like this in Poland. If the Wiki is correct, you don’t even celebrate St Henry. Pleasd, verify this in the official sources.

File: src/calendars/poland.js [hunk 2]
```diff
@@ -335,17 +335,10 @@ let dates = year => {
...
       "moment": moment.utc({ year: year, month: 6, day: 14 }),
-      "data": {
-        "meta": {
-          "liturgicalColor": LiturgicalColors.RED,
-          "titles": [
-            Titles.MARTYR
-          ]
-        }
-      }
+      "data": {}
```
Comment (tukusejssirs) [inline src/calendars/poland.js]: Why is this empty?

File: src/calendars/poland.js (hunk 1 above)
Comment (jarosz) [reply src/calendars/poland.js]: Polish calendar got changed in 2018, I'll have to update the wiki according to https://opoka.org.pl/biblioteka/T/TA/TAL/kkbids/Kalendarz_diecezji_polskich_2018.pdf and review whole poland.js

Comment (tukusejssirs) [reply src/calendars/poland.js]: Okay then, you know it better. :smiley:

File: src/calendars/poland.js (hunk 2 above)
Comment (jarosz) [reply src/calendars/poland.js]: Same reason as above.

**Pull request merged**